cocos/serialization/deserialize-dynamic.ts

```diff
@@ -253,7 +253,7 @@ function compileDeserializeNative (_self: _Deserializer, klass: CCClassConstruct
             if (prop === undefined) {
                 continue;
             }
-            if (!fastMode && typeof prop !== 'object') {
+            if (typeof prop !== 'object') {
```

Comment:
why remove this fastMode

Reply:
There is a problem with not removing this serializatio

Reply:
What problem?